examples/kafka/kafka-cruise-control.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: kafka.strimzi.io/v1beta1
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  kafka:
+    version: 2.4.0
+    replicas: 3
+    listeners:
+      plain: {}
+      tls: {}
+    config:
+      offsets.topic.replication.factor: 3
+      transaction.state.log.replication.factor: 3
+      transaction.state.log.min.isr: 2
+      log.message.format.version: "2.4"
+    storage:
+      type: ephemeral
+  zookeeper:
+    replicas: 3
+    storage:
+      type: ephemeral
+  entityOperator:
+    topicOperator: {}
+    userOperator: {}
+  cruiseControl:
```

Comment:
So out of curiosity ... when will it do when it is deployed like this withotu any other configuration?

Reply:
The k8s yaml validation will throw an error

Now that I have remove `cruiseControl.replicas` we can bring Cruise Control up with just `cruiseControl: {}`